flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/runtime/batch/sql/PartitionableSinkITCase.scala

```diff
@@ -83,9 +79,12 @@ class PartitionableSinkITCase extends BatchTestBase {
         "insert into sinkTable select a, max(b), c"
           + " from nonSortTable group by a, c")
       .await()
-    assertEquals(List("1,5,Hi", "1,5,Hi01", "1,5,Hi02"), RESULT1.sorted)
-    assert(RESULT2.isEmpty)
-    assertEquals(
+    assertThatIterable(RESULT1.sorted).containsExactlyElementsOf(
```

Comment:
probably `containsExactlyInAnyOrderElementsOf` could be used here 
so no sorting is requied

Reply:
Fixed.